### PR TITLE
x11-libs/libva-intel-media-driver and media-libs/gmmlib version bump

### PR DESCRIPTION
**Pull request by Karlson2k**
Updated x11-libs/libva-intel-media-driver to version 21.2.2_pre
Updated media-libs/gmmlib to version 21.1.3

Updated patch for libva-intel-media-driver.

### CODE REVIEW

**Comment by Karlson2k** [comment]
@thesamesam Could you please check this PR?

File: x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	: ${EGIT_REPO_URI:="https://github.com/intel/media-driver"}
+	if [[ ${PV%9999} != "" ]] ; then
+		: ${EGIT_BRANCH:="release/${PV%.9999}"}
+	fi
+	inherit git-r3
+else
+	MY_PV="${PV%_pre}"
+	SRC_URI="https://github.com/intel/media-driver/archive/intel-media-${MY_PV}.tar.gz"
+	S="${WORKDIR}/media-driver-intel-media-${MY_PV}"
+	KEYWORDS="~amd64"
```
Comment (thesamesam) [inline x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild:18]: Let's not add keywords for _pre releases?

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild (hunk 1 above)
Comment (Karlson2k) [reply x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild:18]: Actually any tagged version of Intel Media Driver is stable enough and is more like "RC", but officially they are named sometimes "-pre".
They are not a random snapshot of git, they are stable enough to try. FreeBSD already updated their ports.
As many packages have ebuilds for RC, beta or even alpha keyworded as "unstable", I think it makes sense to use unstable keywords for Intel Media Driver _pre releases, but never stabilize them.

Comment (thesamesam) [reply x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild:18]: Well, I'd still prefer to not keyword release candidates.

In general, I don't really want to put things in ~arch if they're not a stable candidate.

Comment (Karlson2k) [reply x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild]: Keyword removed.

**Comment by Karlson2k** [comment]
@thesamesam Done as requested.

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**